lib/ecto/migration.ex

```diff
@@ -658,4 +659,17 @@ defmodule Ecto.Migration do
   defp validate_type!(%Reference{} = reference) do
     reference
   end
+
+  def prefix(opts_prefix) when is_nil(opts_prefix) do
```

Comment:
Can we not call it from tests? It is an implementation detail, we can probably test this indirectly via `create table` and friends, what do you think?